### ISSUE TITLE
StackedBarChart - when first dataset has zeros, whole bar doesn't show

### ISSUE THREAD

**Issue by etskelly** [FVANCOP/ChartNew.js#228]
When the first dataset has a zero the stacked bar doesn't show at all, surely it should show with the stacks that have values above zero. An example is shown below:

```javascript
var chartData = {
        labels: ["January", "February", "March", "April", "May", "June", "July"],
        datasets: [{
            fillColor: "rgba(133,214,70,0.5)",
            data: [1, 0, 0, 0, 0, 0, 0]
        }, {
            fillColor: "rgba(36,198,232,0.5)",
            data: [1, 2, 2, 1, 1, 0, 0]
        }, {
            fillColor: "rgba(255,212,63,0.5)",
            data: [0, 1, 1, 1, 1, 3, 2]
        }, {
            fillColor: "rgba(255,42,26,0.5)",
            data: [1, 0, 0, 1, 1, 0, 1]
        }]
    }
    window.onload = function () {
        var ctx = document.getElementById("myChart").getContext("2d");
        window.myBar = new Chart(ctx).HorizontalStackedBar(chartData, {
            spaceLeft: 15
        });
    };
```

![screen shot 2015-01-29 at 09 39 06](https://cloud.githubusercontent.com/assets/172249/5955449/d8dc982a-a79a-11e4-870e-0b6a3a6fea04.png)

 So here only January shows, since it has a '1' in the dataset, but the others do not. It should be showing the rest of the bar for the other months.

**Comment by FVANCOP** [comment]
I think that the problem was introduced with the fix for #205.

I have rewritten the fix for issue #205; The new version should solve both problems.

**Comment by etskelly** [comment]
Seems to have done the trick, thanks very much.
